keyboards/shuguet/shu89/keyboard.json

```diff
@@ -0,0 +1,324 @@
+                    {"label": "Space", "matrix": [5, 9], "x": 5, "y": 5.5, "w": 2.25},
+                    {"label": "Space", "matrix": [11, 1], "x": 10, "y": 5.5, "w": 2.75},
+                    {"label": "Alt", "matrix": [11, 3], "x": 12.75, "y": 5.5, "w": 1.25},
+                    {"label": "Win", "matrix": [11, 4], "x": 14, "y": 5.5, "w": 1.25},
+                    {"label": "Menu", "matrix": [11, 5], "x": 15.25, "y": 5.5, "w": 1.25},
+                    {"label": "Ctrl", "matrix": [11, 6], "x": 16.5, "y": 5.5, "w": 1.25},
+                    {"label": "\u2190", "matrix": [11, 7], "x": 18, "y": 5.5},
+                    {"label": "\u2193", "matrix": [11, 8], "x": 19, "y": 5.5},
+                    {"label": "\u2192", "matrix": [11, 9], "x": 20, "y": 5.5}
+                    ]
+        }
+    },
+    "rgb_matrix": {
+        "driver": "ws2812",
+        "split_count": [ 36, 53 ],
+        "default": {
+            "hue": 85,
+            "sat": 255,
+            "val": 128
+        },
+        "animations": {
+            "solid_color": true,
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "rainbow_moving_chevron": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "flower_blooming": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "typing_heatmap": true,
+            "digital_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true,
+            "starlight": true,
+            "starlight_dual_hue": true,
+            "starlight_dual_sat": true,
+            "riverflow": true,
+            "effect_max": true
+        },
+        "layout": [
```

Comment:
These coordinates do not look correct
> The default expected range of values for `{ x, y }` is the inclusive range `{ 0..224, 0..64 }`.

The `x`/`y` values are not the matrix positions.

Reply:
First of all, thank you for the review!
Regarding the coordinate, based on what I read in the docs, the x & y coordinates are supposed to represent the coordinates of the LED on a 224x64 canvas (which is more or less what you confirmed above), right?

If so, why are the x & y values  I picked here an issue?
I only used a small portion of the allocated space, but is there an issue with doing it this way?

Should I consider the bottom right most key on the keyboard to have x = 224 & y = 64?
There's nothing (I could find) in the docs that seems to imply that.

Reply:
Because animations assume the center is {112, 32}.   And if this isn't configured as such, it will cause issues for several animations that use the center for calculations. 

The center *is* configurable, but generally, you shouldn't need to change it.

Reply:
Thank you for the explanation @drashna !
I have some math to do to keep things proportional then, will do that today if possible, I think it's the last thing blocking this PR.

Reply:
@drashna @waffle87 I've updated the X & T coordinates of all LEDs to make use of the full 224x64 grid,

Does this latest revision work for you?

Reply:
Properly fixed this time (I think). Can I get a re-review / approval @waffle87 ? 🙏